week_3/Binary_Tree/index.js

```diff
@@ -0,0 +1,38 @@
+class Node {
+	constructor(val) {
+		this.val = val;
+		this.left = null;
+		this.right = null;
+	}
+}
+class BinarySearchTree {
+	constructor() {
+		this.root = null;
+	}
+	insert(val) {
+		let newNode = new Node(val);
+		if (!this.root) {
+			this.root = newNode;
+			return this;
+		}
+		let comp = this.root;
```

Comment:
what does "comp" stand for? can we rename this to something more descriptive?

Reply:
"comp" stands for "compare." Because this is a Binary Search Tree, nodes are placed according to whether they're greater or less than the current node. I changed the variable name to "current," figuring that the idea of "comparison" will be conveyed through the actual comparative statements that follow.